test/bdd/features/edv_e2e_api.feature

```diff
@@ -10,10 +10,10 @@ Feature: Using EDV REST API
 
   @e2e
   Scenario: Full end-to-end flow. Create a data vault, store an encrypted document, and then retrieve the encrypted document. Query using an encrypted index.
-    Then  Client sends request to create a new data vault with id "testvault" and receives the vault location "localhost:8080/encrypted-data-vaults/testvault" in response
+    Then  Client sends request to create a new data vault with randomized id
```

Comment:
The new data vault always has a randomized ID by design (now), so you don't need to explain that here... you could say that it receives the vault location in response, however... similar to how "document location" is described below.

Reply:
Ah, just saw that you removed the "received document location in return" text. I think you can keep that, but just remove the specific URL that's specified since now the vaultID is randomly generated, of course

Reply:
fixed